src/cascade/executor/dismod_runner.py

```diff
@@ -153,51 +162,46 @@ def run_and_watch(command, single_use_machine, poll_time):
         str: The output stream.
         str: The error stream.
     """
+    command = [str(a) for a in command]
+    loop = asyncio.get_event_loop()
+    return loop.run_until_complete(_async_run_and_watch(command, single_use_machine, poll_time))
+
+
+@asyncio.coroutine
+def _async_run_and_watch(command, single_use_machine, poll_time):
     if single_use_machine:
         pre_execution_function = reduce_process_priority
     else:
         pre_execution_function = None
 
     try:
         CODELOG.info(f"Forking to {command}")
-        sub_process = subprocess.Popen(
-            args=command,
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
-            universal_newlines=True,
-            preexec_fn=pre_execution_function
-         )
-    except ValueError as ve:
-        raise Exception(
-            f"Dismod called with invalid arguments {ve}"
+        sub_process = yield from asyncio.subprocess.create_subprocess_exec(
+            *command, stdout=asyncio.subprocess.PIPE, stderr=asyncio.subprocess.PIPE, preexec_fn=pre_execution_function
```

Comment:
Took out universal newlines? It was there... why? There was a reason. I should have left a comment about that, and now I can't remember. It turns all line endings into \n, whether they come from PC or from Mac or Linux. Maybe I just liked that idea.

Reply:
From the asyncio docs: "The universal_newlines parameter is not supported (only bytes strings are supported)"

That said, I'm not particularly interested in supporting Windows unless you think it's worth putting effort into.